examples/pytorch/mock_sparse/gcn.py

```diff
@@ -0,0 +1,152 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import dgl
+from dgl.data import CoraGraphDataset, CiteseerGraphDataset, PubmedGraphDataset
+from dgl import AddSelfLoop
+import argparse
+from torch.nn import init
+
+from dgl.mock_sparse import create_from_coo, diag, identity
+
+from synthetic import SyntheticDataset
```

Comment:
Remove SyntheticDataset and the corresponding file.

Reply:
Done